src/player.js

```diff
@@ -560,4 +644,5 @@ class Player extends FakeEventTarget {
 // </editor-fold>
 }
 
-export default Player;
+const registerEngine = Player.registerEngine;
+export {registerEngine};
```

Comment:
Let's remove for now, don't see a use case for external playback engine that we want to support without too much added complexity.